### DISCUSSION TITLE
Please no version number in the name of the exe-file

### DISCUSSION THREAD

**Discussion by Peter-202122** [sumatrapdfreader/sumatrapdf#3952] [General]
Hello,

in the installer version the name of the exe-file is simply SumatraPDF.exe. 

But in the **portable** version its name is for example: SumatraPDF-**3.5.2-64**.exe
I.e. the file name contains also the version number (and the information about 64- or 32-bit build).

This complicates the situation after an update of Sumatra because every time the file name is a different one:
The effect is that after every update I have e.g. to modify my desktop shortcut and to re-establish in Windows the SumatraPDF as my standard app for opening PDF-files. 

Yes, I could rename after every update the file name by myself. But it would be more comfortable if the name was SumatraPDF.exe straight after downloading it. ;-)

Thank you.

**Comment by kjk** [comment]
If you use `Check for updates` command in portable and click `Install and relaunch` in the dialog, sumatra will update itself in-place i.e. the name will not change hence no need to change shortcuts.

**Reply by Peter-202122** [reply]
A, good to know. Thank you. :-)

**Reply by Peter-202122** [reply]
Sorry, I tried it just know, but Sumatra did not update itself. When following the steps you mentioned ("Check for updates" -> "Install and relaunch") only the download page of sumatrapdfreader.org opens in the browser and I have to perform downloading and installation manually.

Anyway, no big problem. If the naming of the exe-files remains as it is, I will modify the name myself after every dowload of a new version.

**Reply by kjk** [reply]
You must be using pretty old version. This functionality wasn't always there but that's how it should work with the latest version.

**Reply by Peter-202122** [reply]
I use the latest version (3.5.2) but as (of course) it cannot offer an update, I tested with 3.4.6 (the version I had used before). 
According to what you say, it will work with 3.5.2 when the next update is ready. :-) Thanks for the clarification.
